src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -91,53 +91,45 @@ internal static bool IsValidPublicKey(ImmutableArray<byte> blob)
                 return false;
             }
 
-            return IsValidPublicKeyUnsafe(blob);
-        }
+            BinaryReader blobReader = new BinaryReader(new MemoryStream(blob.DangerousGetUnderlyingArray()));
```

Comment:
`BinaryReader` and `MemoryStream` implement `IDisposable`. Do we need `using` blocks here?